api/provider/aws/common.go

```diff
@@ -68,6 +68,13 @@ func getLoadBalancerRoleName(loadBalancerID string) string {
 	return fmt.Sprintf("%s-lb", loadBalancerID)
 }
 
+func taskFamilyRevisionFromARN(taskARN string) (string, string) {
```

Comment:
[nit] Since I'm not sure if this is called by anything external to Tasks yet, I have to ask: does this function have to live in `common.go`?

Reply:
I expect it will with deploys and services, but since that isn't the case yet, I'll move it